src/app/public/modules/code-examples/code-examples-editor.service.ts

```diff
@@ -186,7 +183,7 @@ export class AppComponent {
   ) {
     if (themeSvc) {
```

Comment:
Do we need the `if` check anymore?

Reply:
Nope. I made the service a required param, removed the `if` and then fixed the tab/spacing here.